web/html/src/manager/minion/ansible/schedule-playbook.tsx

```diff
@@ -66,7 +68,7 @@ export default function SchedulePlaybook({ playbook, onBack }: SchedulePlaybookP
   const schedule = () => {
     return Network.post("/rhn/manager/api/systems/details/ansible/schedule-playbook", {
       playbookPath: playbook.fullPath,
-      inventoryPath: inventoryPath?.text,
+      inventoryPath: inventoryPath?.text === "-" ? null : inventoryPath?.text,
```

Comment:
Since `"-"` seems to be a magic string, maybe we can lift it out into a const or something similar?